backend/src/exceptions/exception.fillter.ts

```diff
@@ -0,0 +1,28 @@
+import { Catch, ExceptionFilter, ArgumentsHost, HttpException, HttpStatus } from '@nestjs/common';
+import { Response } from 'express';
+import { ValidationException } from './validation.exception';
+
+@Catch(HttpException)
+export class DuplicateEmailExceptionFilter implements ExceptionFilter {
```

Comment:
현재 클래스가 처리해주는게 이메일 중복도 처리해주지만 User라는 Entity에 대해서 유효하지 않은 입력값들에 대해서도 처리를 해주는 역할인 것 같아요! 따라서 클래스명을 아래처럼 수정해주시면 좋을 것 같습니다!
```suggestion
export class UserExceptionFilter implements ExceptionFilter {
```